components/manager/scripts/create.py

```diff
@@ -89,6 +91,46 @@ def normalize_agent_name(filename):
             # ubuntu-trusty-agent
             return filename.split('_', 1)[0].lower()
 
+        def backup_agent_resources(agents_dir):
+            ctx.logger.info('Backing up agents in {0}'.format(agents_dir))
+            utils.mkdir(AGENTS_ROLLBACK_PATH)
+            utils.copy(agents_dir, AGENTS_ROLLBACK_PATH)
+
+        def restore_agent_resources(agents_dir):
+            ctx.logger.info('Restoring agents in {0}'
+                            .format(AGENTS_ROLLBACK_PATH))
+            utils.remove(agents_dir)
+            utils.mkdir(agents_dir)
+            utils.copy(os.path.join(AGENTS_ROLLBACK_PATH, 'agents/.'),
+                       agents_dir)
+
+        manager_resources_home = utils.MANAGER_RESOURCES_HOME
+        manager_agents_path = utils.AGENT_ARCHIVES_PATH
+
+        manager_scripts_path = '{0}/packages/scripts'.format(
+                manager_resources_home)
+        manager_templates_path = '{0}/packages/templates'.format(
+                manager_resources_home)
+        if utils.is_upgrade:
+            ctx.logger.info('Manager is in upgrade state.')
```

Comment:
this is not indicative of what's happening.
I don't think we need all of this logging (beneath as well). Simply stating that you're "Replacing Agents..." and in rollback "Restoring agents..." would be enough